code/computational_geometry/jarvis_march/jarvis_march.py

```diff
@@ -0,0 +1,93 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import math 
+
+class Point:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def equal(self, pt):
+        if self.x == pt.x and self.y == pt.y:
```

Comment:
Just `return self.x == pt.x and self.y == pt.y` needs to be the only line in this function.

Reply:
fixed.